Filter/StringFilter.php

```diff
@@ -25,28 +25,39 @@ public function filter(ProxyQueryInterface $proxyQuery, $alias, $field, $data)
             return;
         }
 
-        $data['value'] = trim($data['value']);
+        $value = $data['value'] = trim($data['value']);
```

Comment:
didn't wanna change the former behavior. Do not know if somebody use the trimmed value afterwards and expects it to be trimmed.